### PR TITLE
chore: add schema-diff script

### PR DESCRIPTION
**Pull request by pluveto**
### Summary

This is a draft. We found it hard to keep schema field migration clear. So I wrote this script for aiding.

### Checklist

- [ ] The Pull Request has tests
- [ ] There's an entry in the CHANGELOG
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Full changelog

* [Implement ...]

### Issue reference

Fix #_[issue number]_

### CODE REVIEW

File: scripts/schema-diff.lua [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    print(cjson_encode({
+        added = result.added,
+        removed = result.removed,
+        changed = result.changed,
+    }))
+end
+
+if arg then
+    entry()
+end
```
Comment (StarlightIbuki) [inline scripts/schema-diff.lua:211]: Newline at EOF.

Comment (pluveto) [reply scripts/schema-diff.lua]: yeah 🤣

**Comment by chronolaw** [comment]
Style: in kong repo, we should have two space indent, not four. See `changelog-helper.lua`

**Comment by StarlightIbuki** [comment]
Is this PR still needed or in progress?

**Comment by pluveto** [comment]
> Is this PR still needed or in progress?

I have no idea

**Pull request closed without merging**